modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,22 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parseAndSummarise(file: str, config: Config,
+                        delete_after_parsing: bool):
+    with open(file, 'rb') as fd:
+        data = fd.read()
+        stream = DataStream(data, byte_order='big', is_signed=False)
+        try:
+            eid, summary = parsePelSummary(stream, config)
+            if(eid):
+                return eid, summary
+            else:
+                print(f"No PEL parsed for {file}")
```

Comment:
How de we get error message and PEL summary both will be printed in the console, right?

Please add test result with negative case as well to check command output.